pithos/plugins/notify.py

```diff
@@ -45,6 +45,14 @@ def on_prepare(self):
         if 'action-icons' in caps:
             self.notification.set_hint('action-icons', GLib.Variant('b', True))
 
+        # GNOME Shell 3.20 or higher has bultin MPRIS functionality that makes
+        # persistent song notifications redundant.
+        min_version = Gtk.get_major_version() >= 3 and Gtk.get_minor_version() >= 20
```

Comment:
> Gtk version doesn't necessarily mean Gnome-Shell version.

I didn't think it was possible to run a mismatched Gtk and GNOME Shell version?

> note that we'll need a way that works within the flatpak sandbox too

I was under the impression that only Gio.Notifications worked in the sandbox?